pvlib/snowcoverage.py

```diff
@@ -0,0 +1,170 @@
+"""
+The ``snow`` module contains functions that model the effect of snow on
+solar modules.
+"""
+
+import numpy as np
+import pandas as pd
+from pvlib.tools import sind
+
+
+def _time_delta_in_hours(times):
+    delta = times.to_series().diff()
+    return delta.dt.total_seconds().div(3600)
+
+
+def snow_nrel_fully_covered(snowfall, threshold=1.):
```

Comment:
if the module name is `snow`, then we can drop `snow_` from the function names (my preference, as discussed above). Obviously they will need to stay if the functions move to `losses.py`.

Reply:
I agree.

Do we want to hold this PR until we bring the losses discussion to a conclusion?  OK with me to wait here.